.gitignore

```diff
@@ -6,6 +6,11 @@
 .vscode/
 .nova/
 
+# Version Managers
+.tool-versions
+.nvmrc
```

Comment:
Do not remove `.nvmrc` is required for `nvm use` command to work.

Reply:
Never heard of `.tool-version` before, but I use `.nvmrc` often – local development work well only with Node 14 or 16, while 18 produces errors.

Reply:
`.tool-versions` is for [asdf](https://github.com/asdf-vm/asdf). It's basically an all-in-one tool for programming language version managers. Personally, I use it to manage versions of node, rust, and python.

Since node 18 has issues in development, I'll revert the commit and just synchronize the two files.

Reply:
I probably should have just put that in a separate PR, but ah well.